legislation.tex

```diff
@@ -67,7 +67,7 @@ \chapter{Cake invoking circumstances}\label{chp:circumstances}
     \emph{COR}.
   \item Using incorrect recipient in sufficiently important email.
   \item Having passed an exam, either final exam or getting the top grade.
-  \item Extreme weather (\emph{cake responsible organization} is Clearhaus A/S).
+  \item Extreme weather (\emph{cake responsible organization} is Clearhaus A/S).\footnote{As defined by DMI as a Category 3 warning in the Aarhus area.}
```

Comment:
Perhaps `s/warning/Warning/`?

How about `s/3/2/`? :slightly_smiling_face: Aarhus is almost never hit by very extreme weather...

http://www.dmi.dk/vejr/tjenester/varsler/

Reply:
Not sure what the rest think of lowering to 2

Reply:
I approve of category 2 :+1: